customize_fusion_values.yaml.example

```diff
@@ -222,20 +222,21 @@ templating:
 # config-sync-github-oauth-token
 # for subscriber mode, enable pulsar and use the "sub" profile
 # springProfilesOverride: "kubernetes,jwt,fusion,sub", see below
-config-sync:
-  enabled: false
-  logstashEnabled: false
-  pub:
-    git:
-      repo: TODO_GITHUB_REPO_URL
-      alias: fusion-config-sync
-      branch: stage
-    github:
-      username: TODO_GITHUB_USERNAME
-      email: TODO_GITHUB_EMAIL
-  pulsar:
-    enabled: false
-  springProfilesOverride: "kubernetes,jwt,fusion,pub"
+#As of July, 2021, config-sync only used internally, not for customer #documentation or use.
```

Comment:
Why hashtag before documentation?

Reply:
The yaml still validates, but it just seemed out of place.

Reply:
Removed the # in front of the word documentation. Originally there because it had wrapped into another line.